backend/src/main/java/com/beaplab/BeaplabEngine/service/AppleWatchService.java

```diff
@@ -37,7 +38,7 @@
 @Service("appleWatchService")
 @ComponentScan("com.beaplab.BeaplabEngine")
 @PropertySource("classpath:r_repo.properties")
-public class AppleWatchService {
+public class AppleWatchService implements WatchService{
```

Comment:
I think thats all that needs to be done for the interface

Reply:
yup pretty much